### ISSUE TITLE
Sound & Video preview requirements?

### ISSUE THREAD

**Issue by azirer** [blueimp/jQuery-File-Upload#3338]
Version 9.7.1 on localhost with latest xampp, the sound preview only works before the upload but not as soon as the file is on the server. 
Video preview doesn't work anywhere either before upload or after that for me. 
On both cases instead of the preview I get  an empty  
`<td><span class="preview"></span></td>`

what should I look at? Is there any place I can see video/sound preview working? Image preview is working fine.

**Comment by blueimp** [comment]
This is an old issue that got caught in the closure of the issue tracker, but I'll answer it now:

Audio&Video previews are only implemented for the upload part (this project focuses on the upload process).
The requirements for displaying audio&video are not well documented, but are visible in the source code:
https://github.com/blueimp/jQuery-File-Upload/blob/master/js/jquery.fileupload-audio.js#L62
https://github.com/blueimp/jQuery-File-Upload/blob/master/js/jquery.fileupload-video.js#L62

Basically, the requirements are support for HTML5 video/audio and a file type that the browser can play.
